From 4b12cf779282c51ae6b71b17b40ac99c24e9c432 Mon Sep 17 00:00:00 2001
From: linmin <linmin@eswincomputing.com>
Date: Fri, 17 May 2024 13:06:31 +0800
Subject: [PATCH 036/109] WIN2030-15279:feat(web-server.c):Add power lost
 resume callback

Changelogs:
1.Add callback for get_power_lost_resume_attr()
2.Add callback for change_power_lost_resume_attr()

Change-Id: I5d635d1c421c6ce39247adf297e882eacb37d001
---
 Core/Src/web-server.c | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/Core/Src/web-server.c b/Core/Src/web-server.c
index 05f9cb7..eef39d2 100644
--- a/Core/Src/web-server.c
+++ b/Core/Src/web-server.c
@@ -10172,12 +10172,10 @@ static int change_power_status(int status)
 }
 
 int get_power_lost_resume_attr(){
-	printf("TODO call get_power_lost_resume_attr \n");
-	return 1;//1:powerON,0:powerOFF
+	return is_som_pwr_lost_resume();
 }
 int change_power_lost_resume_attr(int status){//status 0:power off,1:power on
-	printf("TODO call change_power_lost_resume_attr %d \n",status);
-	return 0;//0 success
+	return es_set_som_pwr_lost_resume_attr(status);
 }
 
 int reset()
-- 
2.25.1

